### DISCUSSION TITLE
Confusion about comment on Project 1, Question 1c.

### DISCUSSION THREAD

**Discussion by ilh21001** [cooperrc/computational-mechanics#231] [Q&A]
Hi, so I’m a bit confused about what I should fix regarding Project 1 Question 1c, as the import datetime and timedelta are highlighted but I’m unsure how I should go about fixing it. From digging into older videos, I found my time to be close to the predicted time but not exact. Attached in the comment in question. 
![IMG_0317](https://github.com/cooperrc/computational-mechanics/assets/154881137/ee4f9897-e27d-4ee0-9677-9ccce78670c7)

**Comment by cooperrc** [comment]
Comments get hidden by classroom for some reason. If you open in a new tab you can see the comment with that box.
